resources/aws_ec2.go

```diff
@@ -641,11 +618,14 @@ func (obj *AwsEc2Res) longpollWatch() error {
 			}
 		case msg, ok := <-obj.awsChan:
 			if !ok {
-				return *exit
+				return fmt.Errorf("channel closed unexpectedly")
```

Comment:
If you want me to put this in another patch I don't mind. It came up when I was testing and I hadn't noticed.

Reply:
Should this be an error or nil?